docs/application/tizen-studio/TOC.md

```diff
@@ -114,14 +115,14 @@
 
```

Comment:
Current file name for platform-tools/overview.md is platform-tools/cover-platform.md.